src/oapv_tpool.c

```diff
@@ -679,7 +679,7 @@ int oapv_tpool_spinlock_wait(volatile int *addr, int val)
     return temp;
 }
 
-void threadsafe_assign(volatile int *addr, int val)
+void threadsafe_assign_oapv(volatile int *addr, int val)
```

Comment:
It is recommended to use 'static' function rather than adding suffix ('_oapv').
And the the symbol would not be exported to explicitly.

Reply:
GCC won't accept that. Unless it's static in xeve and openapv. From what I see xeve is no longer updated.

Reply:
When I checked the code, it found the threadsafe_assign() function is not used in openapv.
I think the best way is to remove the fuction in this time.